tests/integration_tests/functional/test_logging.py

```diff
@@ -55,6 +55,7 @@ def check_log_message_format(log_str, instance_id, level, show_level,
     strptime(timestamp, "%Y-%m-%dT%H:%M:%S")
```

Comment:
The description for this function should also be updated to match the new addition to the log format.